tests/ounit/utils/testGraph.ml

```diff
@@ -0,0 +1,58 @@
+(* This file is part of the Kind 2 model checker.
+
+   Copyright (c) 2020 by the Board of Trustees of the University of Iowa
+
+   Licensed under the Apache License, Version 2.0 (the "License"); you
+   may not use this file except in compliance with the License.  You
+   may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0 
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   implied. See the License for the specific language governing
+   permissions and limitations under the License. 
+
+ *)
+(** Testing the graph implementation
+   
+   @author Apoorv Ingle *)
+
+open OUnit2
+
+module G = Graph.Make(struct
+               type t = LustreAst.ident
+               let compare = Stdlib.compare
+               let pp_print_t = LustreAst.pp_print_ident 
+             end)
+
+let v0 = "v0"
+let v1 = "v1"
+                         
+let singleton_g = G.add_vertex G.empty v0
+let dos_g = G.add_vertex singleton_g v1
+
+let dos_connected_g = G.add_edge dos_g (G.mk_edge v0 v1)
+let dos_cycle_g = G.add_edge dos_connected_g (G.mk_edge v1 v0)
+
+let g_verts = List.fold_left (G.add_vertex) G.empty ["t0";"t1";"t2";"int"; "c"; "1"]
+let g = List.fold_left (G.add_edge) g_verts [G.mk_edge "t0" "t1"
+                                       ; G.mk_edge "t1" "t2"
+                                       ; G.mk_edge "t2" "int"
+                                       ; G.mk_edge "c" "t0" 
+                                       ; G.mk_edge "c" "1" ]
```

Comment:
`g_verts` and `g` are not used. It would be good to have a test with them.

Reply:
We check this in `TestGraphAdapter` I have removed this now.